masschallenge/django-accelerator
# [AC-9467]: Make Startup fields nullable

[AC-9467](https://masschallenge.atlassian.net/browse/AC-9467)

How to test:
- On your local machine, run migrations
- Check that the def of done according to [AC-9467](https://masschallenge.atlassian.net/browse/AC-9467) is satisfied
